packages/app-mobile/components/voiceTyping/AudioRecordingBanner.tsx

```
@@ -127,29 +124,18 @@ const useAudioRecorder = (onFileSaved: OnFileSavedCallback, onDismiss: ()=> void
+			
 			setRecordingState(RecorderState.Recording);
-			const recording = new Audio.Recording();
-			await recording.prepareToRecordAsync(recordingOptions());
-			recording.setOnRecordingStatusUpdate(status => {
-				setDuration(status.durationMillis);
-			});
-			recordingRef.current = recording;
-			await recording.startAsync();
+			await recording.prepareToRecordAsync();
+			setDuration(recording.currentTime * 1000);
+		recordingRef.current = recording;
```

ESLint

[error] Expected indentation of 3 tabs but found 2.

(`@typescript-eslint/indent`)

---

[error] Expected indentation of 3 tabs but found 2.

(indent)